MetaCell/cloud-harness
# CH-142 Add dockerfile build args support to apps

Closes [CH-142](https://metacell.atlassian.net/browse/CH-142)

# Implemented solution

A `dockerfile` section has been added under the `haress` section of an application which then has a `buildArgs` map which is provided to the dockerfile by Skaffold when building.

# How to test this PR

Either run `pytest` to run the added unit test, or run `harness-deployment -i samples` and `skaffold dev` to see the value set in the values echoed out during build.

# Sanity checks:

# Breaking changes (select one):

# Possible deployment updates issues (select one):

### Test coverage (select one):

### Documentation (select one):

### Nice to have (if relevant):


[CH-142]: https://metacell.atlassian.net/browse/CH-142?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
